src/components/debug-panel/debug-panel.tsx

```diff
@@ -1,337 +1,222 @@
+import { ReactEventHandler } from "react";
 import styled from "styled-components";
 
-import { DebugOptionGroup, ToggleSwitch, Checkbox } from "../";
-
-import { Color, Font, DropDownStyle } from "../../constants/common";
-
 import {
-  TILE_COLOR_MODES,
-  BOUNDING_VOLUME_COLOR_MODES,
-  BOUNDING_VOLUME_TYPE,
-} from "../../constants/map-styles";
```

Comment:
Cleanup removed resources it they are not used anywhere else

Reply:
Cleanup some data.